### DISCUSSION TITLE
Thanks a lot for this great work

### DISCUSSION THREAD

**Discussion by harshitmaurya25** [fief-dev/fief#333] [General]
Thanks a lot for this amazing work !!!!!

**Comment by fief-bailiff** [comment]
Hail, @harshitmaurya25 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
I see you edited your question, so you probably found the solution :) But in any case: you need to explicitly allow redirect URL in your Client: https://docs.fief.dev/configure/clients/#redirect-uris
